lib/block/gfm/table/index.js

```diff
@@ -168,7 +168,9 @@ class Table extends Parent {
         const diffs = diff(oldValue, value)
         const { path } = cell
         path.push('meta', 'align')
-        this.jsonState.pushOperation('editOp', path, 'text-unicode', diffToTextOp(diffs))
+        if (diffToTextOp(diffs).length > 0) {
```

Comment:
```js
  alignColumn (offset, value) {
    const { columnCount } = this
    if (offset < 0 || offset >= columnCount) {
      debug.warn(`Column at ${offset} is not existed.`)
    }
    console.log(offset, value)
    const table = this.firstChild
    table.forEach(row => {
      const cell = row.find(offset)
      if (cell) {
        const { align: oldValue } = cell
        cell.align = oldValue === value ? 'none' : value
        // dispatch change to modify json state
        const diffs = diff(oldValue, cell.align)
        const { path } = cell
        path.push('meta', 'align')
        this.jsonState.pushOperation('editOp', path, 'text-unicode', diffToTextOp(diffs))
      }
    })
  }
```

We need to diff the oldValue and the newValue, and the new value is `cell.align`